recipes/genomepy/meta.yaml

```diff
@@ -49,8 +49,11 @@ test:
   imports:
     - genomepy
 
+  commands: 
+    - genomepy -h # [not py3k]
```

Comment:
If you use
```yaml
extra:
  container:
    # click requires a unicode locale when used with Python 3
    # extended-base generates en_US.UTF-8 locale and sets LC_ALL, LANG properly
    extended-base: true  # [py3k]
```
, you can re-enable this test.

Reply:
This is very useful to know, thanks @mbargull!